File: src/ELibrary.Backend/ShopApi/Features/StatisticsFeature/Domain/Models/GetBookStatistics.cs
using LibraryShopEntities.Domain.Entities.Library;

namespace ShopApi.Features.StatisticsFeature.Domain.Models
{
    public class GetBookStatistics
    {
        public DateTime? FromUTC { get; set; }
        public DateTime? ToUTC { get; set; }
        public Book[] IncludeBooks { get; set; }
    }
}

File: src/ELibrary.Frontend/src/app/modules/shared/domain/models/shop/statistics/getBookStatistics.ts
import { Book } from "../../../..";

export interface GetBookStatistics {
    fromUTC: Date | null,
    toUTC: Date | null,
    includeBooks: Book[]
}

export function getDefaultGetBookStatistics(): GetBookStatistics {
    return {
        fromUTC: null,
        toUTC: null,
        includeBooks: []
    }
}